apps/src/music/blockly/extensions.js

```diff
@@ -151,3 +153,13 @@ export const effectsFieldExtension = function () {
     baseHandler.call(fieldEffectsName, menu, menuItem);
   };
 };
+
+export const fieldSoundsValidator = function () {
+  this.getField('sound').setValidator(newValue => {
```

Comment:
minor: can use the `FIELD_SOUNDS_NAME` constant here to keep it consistent with the field definition

Reply:
Oh good call! I meant to check to see if I could find a constant for this. Thanks for helping me find it.